### PR TITLE
[fix] zoom shortcuts

### PR DESCRIPTION
**Pull request by steveruizok**
This PR overrides the default keyboard shortcuts for zoom in, out, and reset in the VS Code extension.

It also adds the numpad keys to the app's shortcuts for zoom in, out, and reset.

### CODE REVIEW

**Comment by steveruizok** [comment]
Note that the new commands have no effect when selected from the menu, they serve only to prevent the default behavior. I'm sure not how to solve for both cases.

**Comment by seflless** [comment]
Nice PR! The code looks good. 
 - Would prefer to run it locally to see it in action before approving. I think I’ll have time today
 - It’d be great to get the menu triggered one working too. If you don’t take a crack at that, I can. Fine to merge without it, but a bit more legwork maybe first. Let’s get this in today

**Comment by seflless** [comment]
What was the previous behavior? Not working at all? Is there any related issue filed for this?

**Comment by seflless** [comment]
Hmmmm. Zoom at the system level is supposed to be more accessibility zoom, aka font size increasing. So I think not trying to override the menu Zoom- in/out/reset might be best.

File: apps/vscode/extension/package.json [hunk 1]
```diff
@@ -51,11 +51,80 @@
...
+		"keybindings": [
+			{
+				"key": "cmd+numpad_add",
+				"title": "Zoom In",
+				"command": "tldraw.tldr.zoomIn",
+				"category": "tldraw",
+				"enablement": "resourceExtname == .tldr"
+			},
+			{
+				"key": "shift+cmd+=",
```
Comment (seflless) [inline apps/vscode/extension/package.json:63]: This one doesn't work for me.

**Review by seflless — CHANGES_REQUESTED** [review]

File: apps/vscode/extension/package.json (hunk 1 above)
Comment (steveruizok) [reply apps/vscode/extension/package.json:63]: We override this one but we don't actually use it in the app. So maybe we shouldn't override it.

File: apps/vscode/extension/package.json [hunk 2]
```diff
@@ -51,11 +51,80 @@
...
+				"command": "tldraw.tldr.zoomOut",
+				"category": "tldraw",
+				"enablement": "resourceExtname == .tldr"
+			},
+			{
+				"key": "cmd+numpad0",
+				"title": "Zoom Out",
+				"command": "tldraw.tldr.zoomReset",
+				"category": "tldraw",
+				"enablement": "resourceExtname == .tldr"
```
[the remark this answers is not in the document]
Comment (steveruizok) [reply apps/vscode/extension/package.json:102]: Shift+0, unfortunately Command+0 can't be stopped in the browser.

**Review by seflless — APPROVED** [review]
Looks good to me.

**Comment by steveruizok** [comment]
Ok, got it now. Turns out we don't need the menu commands, which solves the "what should they do" problem. Those commands are now just noops as far as VS Code is concerned, and the app itself will take care of the behaviour.

**Pull request merged**